tests/test_rpc.py

```diff
@@ -1188,9 +1215,7 @@ def test_rbfpsbt_cancel(lianad, bitcoind):
     # But we can't set the feerate explicitly.
     with pytest.raises(
         RpcError,
-        match=re.escape(
-            "A feerate must not be provided if creating a cancel."
-        ),
+        match=re.escape("A feerate must not be provided if creating a cancel."),
```

Comment:
these lines have been reformated by black

Reply:
You could commit those lines separately, but I think it's also fine as part of the same commit.